### PR TITLE
Copy edit: angular/guide/template-syntax.md

### PR DESCRIPTION
**Pull request by 100percentlinen**
I only managed to edit up to line 781. Overall, the content conformed pretty well to the Google dev style guide. And I didn't find any typos!

### CODE REVIEW

File: src/angular/guide/template-syntax.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ it assigns this composite interpolated result to an **element or directive prope
 
 You appear to be inserting the result between element tags and assigning it to attributes.
 It's convenient to think so, and you rarely suffer for this mistake.
-Though this is not exactly true. Interpolation is a special syntax that Angular converts into a
+However, it's not exactly true. Interpolation is a special syntax that Angular converts into a
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: How about this rewording of the paragraph:

From a quick glance at the syntax, it looks as if you're inserting the result between element tags and assigning it to attributes. That's a convenient way to think of what's happening, but it's not exactly true. Interpolation is a special syntax that Angular converts into a [property binding](#property-binding). See the details [below](#property-binding-or-interpolation).

File: src/angular/guide/template-syntax.md [hunk 2]
```diff
@@ -209,46 +209,46 @@ the expression context.
...
-Please follow these guidelines:
+Follow these guidelines:
 
 * [No visible side effects](#no-visible-side-effects)
 * [Quick execution](#quick-execution)
 * [Simplicity](#simplicity)
 * [Idempotence](#idempotence)
 
-The only exceptions to these guidelines should be in specific circumstances that you thoroughly understand.
+The only exceptions to these guidelines are specific circumstances that you thoroughly understand.
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: How about this:

If you make an exception to the above guidelines, make sure you thoroughly understand the circumstances.

File: src/angular/guide/template-syntax.md [hunk 3]
```diff
@@ -209,46 +209,46 @@ the expression context.
...
 
-A template expression should not change any app state other than the value of the
+A template expression must not change any app state other than the value of the
 target property.
 
 This rule is essential to Angular's "unidirectional data flow" policy.
-You should never worry that reading a component value might change some other displayed value.
-The view should be stable throughout a single rendering pass.
+It ensures that reading a component value doesn't change some other displayed value.
+The view is stable throughout a single rendering pass.
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: I think "is" should be "must be".

File: src/angular/guide/template-syntax.md [hunk 4]
```diff
@@ -209,46 +209,46 @@ the expression context.
...
+Expressions need to finish quickly or the user experience may drag, especially on slower devices.
 Consider caching values when their computation is expensive.
 
 #### Simplicity
 
-Although it's possible to write quite complex template expressions, you should avoid them.
+Although it's possible to write quite complex template expressions, it's best to avoid them.
 
-A property name or method call should be the norm.
+A property name or method call is the norm.
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: How about changing the sentence as follows:

In most situations, use a property name or method call.

File: src/angular/guide/template-syntax.md [hunk 5]
```diff
@@ -342,15 +342,13 @@ you're ready to learn about the varieties of data binding syntax beyond interpol
...
-While you could push values to and pull values from HTML,
-the app is easier to write, read, and maintain if you turn these chores over to a binding framework.
-You simply declare bindings between binding sources and target HTML elements and let the framework do the work.
+While you can push values to and pull values from HTML,
+the app is easier to write, read, and maintain if you use a binding framework.
+You declare bindings between binding sources and target HTML elements and then let the framework do the work.
 
-Angular provides many kinds of data binding.
-This guide covers most of them, after a high-level view of Angular data binding and its syntax.
+Below is a high-level summary of Angular data binding and its syntax. It's followed by more detailed information about most of the data binding types that Angular provides.
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: In general, we aim for a line length of 80 characters or fewer.  A bit of deviation is OK (there are many longer lines in this file) but this particular line is quite long. Please split it into 3 lines like this:

Below is a high-level summary of Angular data binding and its syntax. It's
followed by more detailed information about most of the data binding types that
Angular provides.

File: src/angular/guide/template-syntax.md [hunk 6]
```diff
@@ -342,15 +342,13 @@ you're ready to learn about the varieties of data binding syntax beyond interpol
...
 
-Binding types can be grouped into three categories distinguished by the direction of data flow:
-from the _source-to-view_, from _view-to-source_, and in the two-way sequence: _view-to-source-to-view_:
+Binding types can be grouped into three categories based on the direction of data flow: _source-to-view_, _view-to-source_, and two-way sequence _view-to-source-to-view_.
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: In general, we aim for a line length of 80 characters or fewer. A bit of deviation is OK (there are many longer lines in this file) but this particular line is quite long. Please split it into 3 lines like this:

Binding types can be grouped into three categories based on the direction of
data flow: _source-to-view_, _view-to-source_, and two-way sequence
_view-to-source-to-view_.

File: src/angular/guide/template-syntax.md [hunk 7]
```diff
@@ -422,11 +420,9 @@ Then you learn about data binding. The first binding you meet might look like th
 
 You'll get to that peculiar bracket notation in a moment. Looking beyond it,
 your intuition suggests that you're binding to the button's `disabled` attribute and setting
-it to the current value of the component's `isUnchanged` property.
+it to the current value of the component's `isUnchanged` property. Your intuition is incorrect! 
 
-Your intuition is incorrect! Your everyday HTML mental model is misleading.
-In fact, once you start data binding, you are no longer working with HTML *attributes*. You aren't setting attributes.
-You are setting the *properties* of DOM elements, components, and directives.
+Your everyday HTML mental model is misleading. In fact, once you start data binding, you're no longer working with HTML *attributes*. You aren't setting attributes; you're setting the *properties* of DOM elements, components, and directives.
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: In general, we aim for a line length of 80 characters or fewer. A bit of deviation is OK (there are many longer lines in this file) but this particular line is quite long. Please split it into 4 lines like this

Your everyday HTML mental model is misleading. In fact, once you start data
binding, you're no longer working with HTML *attributes*. You aren't
setting attributes; you're setting the *properties* of DOM elements, components,
and directives.

File: src/angular/guide/template-syntax.md [hunk 8]
```diff
@@ -658,7 +654,7 @@ If the name fails to match a property of a known directive or element, Angular r
 
 ### Avoid side effects
 
-As mentioned previously, evaluation of a template expression should have no visible side effects.
+As mentioned previously, the evaluation of a template expression has no visible side effects.
```
Comment (sarahmaddox) [inline src/angular/guide/template-syntax.md]: I think "has" should be "must have".

**Comment by kwalrath** [comment]
@100percentlinen will you be able to make the fixes that @sarahmaddox has suggested, or should I do it?

**Comment by kwalrath** [comment]
Also, we'll need a signed CLA before we can accept this PR.

**Comment by 100percentlinen** [comment]
Hi Kathy, I'd prefer not to make the fixes if that's OK.
Elizabeth

**Comment by kwalrath** [comment]
No worries, @sarahmaddox or I will make the fixes. Thanks very much for your contribution!

**Comment by kwalrath** [comment]
Replaced by #1220.

**Pull request closed without merging**